app/src/test/java/com/ikarabulut/BoardTest.java

```diff
@@ -69,4 +69,121 @@ void setRandomGrid() {
         assertFalse(isDeadBoard);
     }
 
-}
\ No newline at end of file
+    @Test
+    @DisplayName("A cell that is dead can be set to alive")
+    void setAliveAt() {
+        Board board = new Board();
+
+        Boolean cellStatus = board.isCellAliveAt(1, 1);
+        assertFalse(cellStatus);
+
+        board.setAliveAt(1, 1);
+        cellStatus = board.isCellAliveAt(1, 1);
+
+        assertTrue(cellStatus);
+
+    }
+
+    @Nested
+    @DisplayName("Tests for getAliveNeighborsCountOf")
+    class GetAliveNeighborsCountOf {
+        public Board board;
+        @BeforeEach
+        void createBoard() {
+            board = new Board(3,3);
+        }
+        @Test
+        @DisplayName("A cell with 0 alive neighbor should return 0 alive neighbor")
+        void getAliveNeighborsCountOf() {
+            int numOfAliveNeighbors = board.getAliveNeighborsCountOf(1,1);
```

Comment:
Not exactly sure what to do (if anything) at the moment, but just wanted to point out that this spec relies on the fact that creating a new instance of a `Board` creates a board with all "dead" cells. If we were to change that, this spec would fail.

One solution would be to have a helper method in the specs where you set every cell to be "dead" and return a new board.

Reply:
Yeah this is a good call. in my next merge to this branch you'll see the setDeadBoard() helper method that is run before every test in this nested test suite.